project/cmake/addons/addons/adsp.basic/adsp.basic.txt

```diff
@@ -1 +1 @@
-adsp.basic https://github.com/AchimTuran/adsp.basic master
+adsp.basic https://github.com/AchimTuran/adsp.basic fix_win32_cmake
```

Comment:
I tried it, but this does not work here on my windows machine. So I leave it "fix_win32_cmake".